app/views/layouts/application.html.erb

```diff
@@ -53,12 +53,16 @@
         </form> -->
         <ul class="nav navbar-nav navbar-right">
           <% if current_user %>
+            <% if current_user.subscribed %>
+              <li><%= link_to current_user.email, account_path %></li>
+            <% else %>
+              <li><a href="/account">Subscribe to Pro</a></li>
+            <% end %>
+              <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
```

Comment:
To confirm, you prefer the html tags within if/else's to be equal indentation? If so, I will change that. Is that convention? I've been taught its more personal preference, and I find it easier to read, but I would rather learn the way seasoned Rails developers expect it to appear. Thanks for the guidance around these topics.